roles/k3s/install/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Enable IPv4 forwarding
+  sysctl:
+    name: net.ipv4.ip_forward
+    value: "1"
+    state: present
+    reload: yes
+
+- name: Enable IPv6 forwarding
+  sysctl:
+    name: net.ipv6.conf.all.forwarding
+    value: "1"
+    state: present
+    reload: yes
+
+- name: "Delete 'k3s' if already present"
+  file:
+    path: /usr/local/bin/k3s
+    state: absent
```

Comment:
Not sure why this is present. The next task should just redownload the file if it's missing or not the same isn't it?
Actually you would need to provide a checksum of that file to avoid re-downloading it.